src/components/Settings/Profile/Profile.vue

```diff
@@ -41,8 +41,8 @@
             dense
             solo
             v-model="selected_namespace"
-            :items="[current_user['k8s_namespace']]"
-            :label="current_user['k8s_namespace']"
+            :items="current_user['k8s_accounts'].map(acc => acc['namespace'])"
```

Comment:
@troeger currently I display only the namespace, but I could change it to service_account:namespce, just like the admin backend, which would you prefer?

Reply:
Namespace is good enough, people anyway never get the idea of service accounts.